jaseci_kit/jaseci_kit/encoders/utils/evaluate.py

```diff
@@ -31,7 +31,7 @@ def get_inference(model, tokenizer, contexts, candidates, train_config):
         batch = tuple(t.to(train_config['device']) for t in batch)
         context_token_ids_list_batch, context_input_masks_list_batch,\
             candidate_token_ids_list_batch, candidate_input_masks_list_batch,\
-            labels_batch = batch
+            _ = batch
```

Comment:
what's going on here?

Reply:
This was when i merged the branch and was trying to push with a change, the labels are not used for evaluations